lib/rocket_pants/client.rb

```diff
@@ -123,7 +123,7 @@ def unpack(object, options = {})
     # @param [Hash, Object] response the response to check errors on
     # @raise [RocketPants::Error] a generic error when the type is wrong, or a registered error subclass otherwise.
     def check_response_errors(response)
-      if !response.is_a?(Hash)
+      if !response.is_a?(HTTParty::Response)
```

Comment:
So Rails 5 requires all the responses to be `HTTParty::Response`? That seems weird.

Reply:
not exactly -- all responses were always `HTTParty::Response` BUT some versions of HTTParty responded true to is_a?(Hash). when I bumped HTTParty I had to update this